datenguide_python/query_builder.py

```diff
@@ -132,20 +226,37 @@ def __init__(self, start_field: Field):
 
     @classmethod
     def regionQuery(
-        cls, region: str, fields: List[Union[str, "Field"]] = []
+        cls,
+        region: str,
+        fields: List[Union[str, "Field"]] = [],
+        default_fields: bool = True,
     ) -> "Query":
         """Factory method to instantiate a Query with a single region through its region id.
 
         Arguments:
             region {str} -- The region id the statistics shall be queried for.
-            fields {List[Union[str, Field]]} -- all fields that shall be returned
+            fields {List[Union[str, Field]]} -- all fields that shall be
+            returned from the query
             for that region. Can either be simple fields (e.g. name)
             or fields with nested fields.
 
         Returns:
             Query -- A query object with region as start Field.
         """
-        return cls(start_field=Field("region", fields, args={"id": '"' + region + '"'}))
+
+        if default_fields:
+            defaults: List[Union[str, "Field"]] = ["id", "name"]
+            fields = defaults + fields
+
+        return cls(
+            start_field=Field(
+                "region",
+                fields,
+                args={"id": '"' + region + '"'},
+                return_type=Query._return_type_region,
```

Comment:
Seems like a lot of effort to dynamically determine the `Query._return_type_region`. I think hard coding would have been ok as well. We could also reuse our own Field functionality
```python
start_field = Field("",returnType='Query').add_field('region')
start_field.add_args({"id":...})
```

Reply:
yes, wasnt sure there... i now hardcoded it